modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Ratings/IRatingRepository.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Volo.Abp.Domain.Repositories;
+
+namespace Volo.CmsKit.Ratings
+{
+    public interface IRatingRepository : IBasicRepository<Rating, Guid>
+    {
+        Task<List<Rating>> GetListAsync(

```

Comment:
Where did you use this? I see it is used by the app service, but the app service method is already not used. If so, remove this method. Such methods are dangerous since it may return thousands (or millions?) entities without any paging which drops the application performance unnecessarily.